README.md

```diff
@@ -96,13 +96,13 @@ none
     <tr>
       <td>EngineVersion</td>
       <td>Redis version</td>
-      <td>5.0.5</td>
+      <td>5.0.6</td>
```

Comment:
is it possible to not change the default and just add `5.0.6` to the list of allowed values and the version map?

Reply:
good idea and done, also added 6.0 version to make available aws supported versions between range 5.0.5 - 6.2